src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.model;
+
+public class Car {
+    private final int carNumber;
+    private int carMovingDistance;
+
+    public Car(int carNumber, int carMovingDistance) {
+        this.carNumber = carNumber;
+        this.carMovingDistance = carMovingDistance;
+    }
```

Comment:
```suggestion
    public Car() {
        this(0)
    }

    public Car(int carMovingDistance) {
        this.carMovingDistance = carMovingDistance;
    }
```

부생성자가 주생성자를 호출하도록 리팩터링하면 더 좋을 것 같습니다. 
외부에서 기본값을 신경 쓰지 않고도 객체를 생성할 수 있게 됩니다.